Lib/test/test_tarfile.py

```diff
@@ -722,6 +722,28 @@ def format_mtime(mtime):
             tar.close()
             os_helper.rmtree(DIR)
 
+    @staticmethod
+    @unittest.mock.patch("tarfile.data_filter", wraps=tarfile.data_filter)
+    @unittest.mock.patch("tarfile.tar_filter", wraps=tarfile.tar_filter)
+    @unittest.mock.patch("tarfile.fully_trusted_filter", wraps=tarfile.fully_trusted_filter)
+    def test_extractall_default_filter(mock_ft_filter: unittest.mock.Mock,
+                                       mock_tar_filter: unittest.mock.Mock,
+                                       mock_data_filter: unittest.mock.Mock):
```

Comment:
Just use a regular method + chained with statements. Easier to read and to understand.

Reply:
Roger, will fix.